tests/probe/t15-specimen.sh

```
@@ -0,0 +1,66 @@
+#
+#
+#
+#
+#
+# ---------------------------------------------------------------- region B ---
+
+write_report() {
+  OUT=$1
+  shift
+  rm -f $OUT
+  for line in $@; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)